mesheryctl/internal/cli/root/system/restart.go

```diff
@@ -66,6 +66,14 @@ func restart() error {
 			return errors.Wrap(err, utils.SystemError("Failed to restart Meshery"))
 		}
 
+		if skipUpdateFlag {
+			log.Info("Skipping Meshery Update...")
+		} else {
+			if err := start(); err != nil {
+				return errors.Wrap(err, utils.SystemError("Failed to restart Meshery"))
+			}
+		}
+
```

Comment:
In the restart command, I guess we need to only set the `skipUpdateFlag` as the actual function is carried out by `start()`. 

There is already a logging in `start.go` that does this, so no additional logic needs to be written here.

Please check line 178 in `start.go` and confirm-

```go
		if skipUpdateFlag {
			log.Info("Skipping Meshery update...")
		} else {
			err := utils.UpdateMesheryContainers()
			if err != nil {
				return errors.Wrap(err, utils.SystemError("failed to update Meshery containers"))
			}
		}
```

Reply:
Alright

Reply:
So, instead of re-writing the whole fn again, what I need to do in such case?

Reply:
@navendu-pottekkat oh yeah. I removed the fn just now and it worked! 😨

Reply:
@alphaX86 What Navendu is saying here is to only set the value of `skipUpdateFlag` to be true with the help of flag, That's all. That'll handle the whole logic.

Reply:
Oooh OK ok

Reply:
I'll change it after lunch